src/osd/OSD.h

```diff
@@ -1944,6 +1944,14 @@ class OSD : public Dispatcher,
     return pg_map.size();
   }
 
+  std::set<int> get_mapped_pools() {
```

Comment:
could you define this method in `OSD.cc`?

Reply:
done